docs/core/preview3/tools/dotnet-sln.md

```diff
@@ -0,0 +1,58 @@
+---
+title: dotnet-sln command | Microsoft Docs
+description: The dotnet-sln command provides a convenient option to add, remove, and list projects in a solution file.
+keywords: dotnet-run, CLI, CLI command, .NET Core
+author: spboyer
+ms.author: spboyer
+ms.date: 02/06/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: e5a72d3e-c14b-4b0a-a978-c5e54a0988c6
+---
+
+# dotnet-sln (Tooling RC4)
+
+dotnet-sln -- Modify a .NET Core solution file.
+
+## Synopsis
```

Comment:
@blackdwarf if we followed a similar pattern we have been using for other commands, this would probably look like this: `dotnet sln [<solution_file_path>] [<add|remove|list>] [--help] [<projects>]`, no?